WeatherForecast/appsettings.json

```diff
@@ -6,5 +6,12 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "Kestrel": {
+    "Endpoints": {
+      "Http": {
+        "Url": "http://0.0.0.0:5000"
```

Comment:
Avoids having to pass `--urls` when running `dotnet run` (which makes instructions consistent regardless of whether you are using a Dev Container or not)